tests/test-cases/Group1-Docker-Commands/1-38-Docker-Exec.robot

```diff
@@ -227,12 +227,15 @@ Concurrent Simple Exec
     Should Be Equal As Integers  ${rc}  0
 
     :FOR  ${idx}  IN RANGE  1  50
-    \   Start Process  docker %{VCH-PARAMS} exec ${id} /bin/ls  alias=exec-simple-%{VCH-NAME}-${idx}  shell=true
+    \   Start Process  docker %{VCH-PARAMS} exec -e idx=${idx} ${id} sh -c 'echo "index is:${idx}";time /bin/ls' alias=exec-simple-%{VCH-NAME}-${idx}  shell=true
 
     :FOR  ${idx}  IN RANGE  1  50
     \   ${result}=  Wait For Process  exec-simple-%{VCH-NAME}-${idx}  timeout=300s
-    \   Should Be Equal As Integers  ${result.rc}  0
-    \   Verify LS Output For Busybox  ${result.stdout}
+    #\   Should Be Equal As Integers  ${result.rc}  0
+    \   Run Keyword If  ${result.rc} == 0  Log To Console  result is good!  ELSE  Log To Console  result is failure.
+    #\   Verify LS Output For Busybox  ${result.stdout}
+    \   ${status}=  Run Keyword And Return Status  Verify LS Output For Busybox  ${result.stdout}
+    \   Run Keyword If  "${status}" == "PASS"  Log To Console  Verify successful!  ELSE  Log To Console  waiting.....  
```

Comment:
2 comments here
1.  if one of process hit issue and we continue the loop, does the case report failure as expected? Do we need to record error_count and after loop to check whether error_count is 0 or not?
2. I think the Log to console info is a little confusing.How about change to 
Run Keyword If  ${result.rc} == 0  Log To Console  rc=0 is expected ELSE  Log To Console  rc should be 0
Run Keyword If  "${status}" == "PASS"  Log To Console  expected result has been found  ELSE  Log To Console   ${status} is not expected data